documentation-site/examples/table-grid/cell-navigation.js

```diff
@@ -191,7 +191,9 @@ export default function() {
                 {...getCellProps(0, primaryRowIndex)}
                 $gridRow={`span ${row[2].length}`}
               >
-                <StyledLink href={row[1]}>{row[0]}</StyledLink>
+                <StyledLink href={String(row[1])}>
```

Comment:
My guess is this is not necessary in the flow-typed files due to it's type inference feature. Do you mind confirming this and related changes are required?

Reply:
I did this because `yarn tsc` fails when it is not wrapped with `String`. Although after checking it again, `tsx` files are the only ones that throw an error. So yeah, its not needed for the flow-typed files.

What would you prefer in this case, `String(row[1])` or `row[1] as string`

Reply:
To be clear, this is a `.js` file and should not have any impact on `yarn tsc`. Can you confirm that `yarn flow` fails if this is not stringified?

Reply:
Yeah, it doesn't have any impacts on `yarn tsc`. Just asking what should i do for the `tsx` files, should i stringify it or just assert the type string.

Reply:
It's fine to leave for the ts files, I would just like to simplify examples if it's not needed. the flow-typed example files are used to create the simple js example files on the docs site and that's the most common action on the site

Reply:
Aight, thanks for letting me know and for the review!